src/main/resources/settings/host/values/strings.xml

```diff
@@ -2,4 +2,12 @@
 <resources>
     <string name="revanced_settings_confirm_user_dialog_title">Do you wish to proceed?</string>
     <string name="revanced_settings_reset">Reset</string>
+    <string name="revanced_settings_import">Import</string>
+    <string name="revanced_settings_import_sb_userid_present">Your settings contain a private SponsorBlock user id.\n\nYour user id is like a password and it should never be shared.\n</string>
```

Comment:
SponsorBlock is it's own patch. Settings patch is unaware about other patches. For that reason it should not mention SponsorBlock here.